src/main/java/com/google/sps/data/AccessSecret.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.data;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import java.io.IOException;
+import java.lang.Throwable;
+
+public class AccessSecret {
+  private String key = null;
```

Comment:
it's null by default, so you don't really need `= null;`

Reply:
done